.github/workflows/lts.yml

```diff
@@ -19,10 +19,10 @@ name: lts_preview
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 0 * * 1"
+    - cron: "0 0 * * *"
```

Comment:
The lts_preview workflow is not fully stable. does it make sense to run this every day? We get emails for failing jobs then every day

Reply:
kk, I am fine with that.